docs/admin/migration/migration.md

```diff
@@ -0,0 +1,36 @@
+# Migration guide of shared components version 4.0.0
+
+## Breaking changes of upgrade MUI v5 -> v7
+
+    When consuming the Shared components from release version 4.0.0, overriding some component props will no longer work as a result of the MUI upgrade.
```

Comment:
Not sure why the indents where added, but they cause a copy/paste format in md, which - I assume - isn't intended.

```suggestion
When consuming the Shared components from release version 4.0.0, overriding some component props will no longer work as a result of the MUI upgrade.
```